components/nav/mobile/offcanvas.js

```diff
@@ -0,0 +1,108 @@
+import { useState } from 'react'
+import { Dropdown, Image, Offcanvas } from 'react-bootstrap'
+import { MEDIA_URL } from '@/lib/constants'
+import Link from 'next/link'
+import { useServiceWorker } from '@/components/serviceworker'
+import { signOut } from 'next-auth/react'
+import { LoginButtons, NavWalletSummary } from '../common'
+import AnonIcon from '@/svgs/spy-fill.svg'
+
+export default function OffCanvas ({ me, dropNavKey }) {
+  const [show, setShow] = useState(false)
+
+  const handleClose = () => setShow(false)
+  const handleShow = () => setShow(true)
+  const { registration: swRegistration, togglePushSubscription } = useServiceWorker()
+
+  const MeImage = ({ onClick }) => me
+    ? (
+      <Image
+        src={me?.photoId ? `${MEDIA_URL}/${me.photoId}` : '/dorian400.jpg'} width='28' height='28'
+        style={{ clipPath: 'polygon(0 0, 83% 0, 100% 100%, 17% 100%)' }}
+        onClick={onClick}
+      />
+      )
+    : <span className='text-muted'><AnonIcon onClick={onClick} width='22' height='22' /></span>
```

Comment:
Image is not updated in footer and offcanvas on upload because `photoId` is missing in the `ME` query.

Patch:

```diff
diff --git a/fragments/users.js b/fragments/users.js
index 8f94c10e..4befc1b9 100644
--- a/fragments/users.js
+++ b/fragments/users.js
@@ -9,6 +9,7 @@ export const ME = gql`
       id
       name
       bioId
+      photoId
       privates {
         autoDropBolt11s
         diagnostics
```